cms/publications/models.py

```diff
@@ -114,6 +115,17 @@ class PublicationPublicationTypeRelationship(models.Model):
     )
 
 
+class ChangelogEntryLink(ChangelogEntry):
+    class Meta:
+        ordering = ["change_date"]
+
+    publication = ParentalKey(
```

Comment:
I don't quite get what the names `publication` here (and `page` on the other class) tie into (but do get what they do)